### PR TITLE
#27500 Prepare Usps module Tests for PHPUnit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php [hunk 1]
```diff
@@ -308,7 +308,7 @@ public function isGirthAllowedDataProvider()
     /**
      * @return MockObject
      */
-    private function getXmlFactory(): MockObject
+    private function getXmlFactory(): \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php]: Please revert this change.

File: app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php [hunk 2]
```diff
@@ -332,7 +332,7 @@ function ($data) {
     /**
      * @return MockObject
      */
-    private function getRateFactory(): MockObject
+    private function getRateFactory(): \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php]: And this

File: app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php [hunk 3]
```diff
@@ -351,7 +351,7 @@ private function getRateFactory(): MockObject
     /**
      * @return MockObject
      */
-    private function getRateMethodFactory(): MockObject
+    private function getRateMethodFactory(): \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php]: And this

File: app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php [hunk 4]
```diff
@@ -372,7 +372,7 @@ private function getRateMethodFactory(): MockObject
     /**
      * @return MockObject
      */
-    private function getHttpClientFactory(): MockObject
+    private function getHttpClientFactory(): \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php]: And this

File: app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php [hunk 5]
```diff
@@ -394,7 +394,7 @@ private function getHttpClientFactory(): MockObject
     /**
      * @return MockObject
      */
-    private function getProductCollectionFactory(): MockObject
+    private function getProductCollectionFactory(): \PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/Usps/Test/Unit/Model/CarrierTest.php]: And this

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by lbajsarowicz** [comment]
@ihor-sviziev Fixed. This was caused by the fact that firstly - the Class import was executed, and just after then replacement of classes.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7380](https://jira.corp.magento.com/browse/ENGCOM-7380) has been created to process this Pull Request
